index.html

```diff
@@ -62,23 +70,28 @@ <h2>My Recent Works</h2>
       </div>
 
       <div class="sample-work">
-        <img src="./assets/images/work-img-1.png" alt="work-1" />
-        <h3>Multi-Post Stories</h3>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul class="tags">
-          <li class="tag-item">css</li>
-          <li class="tag-item">html</li>
-          <li class="tag-item">bootstrap</li>
-          <li class="tag-item">Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
+        <div><img src="./assets/images/work-img-1.png" alt="work-1" /></div>
+        <div class="work-content">
+          <h3>Multi-Post Stories</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="tags">
+            <li class="tag-item">css</li>
+            <li class="tag-item">html</li>
+            <li class="tag-item">bootstrap</li>
+            <li class="tag-item">Ruby</li>
+          </ul>
+          <button type="button">See Project</button>
+        </div>
       </div>
 
       <div class="works-grid">
+        <div class="first work-item">
+          <button type="button">See Project</button>
```

Comment:
Kindly add more projects so that your design represents fully the design given by your template. This will help to show the full layout of the website.

Your design:
![image](https://user-images.githubusercontent.com/20554583/186623115-82129600-bed8-4848-8d3b-767007f254ba.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/186623488-5fa4903f-9a47-40a6-8b82-f28fc5779a17.png)